docs/edges.md

```
@@ -171,6 +171,48 @@ Loop edges are **not** counted as cycles by DIP005 validation — they are the i
+    SetupRun -> Done         on setup-resume-required
+    RunTests -> Package      on tests-ok
+    else -> Cleanup
+```
+
+It exists to collapse the **error funnel** — the dominant line-count cost in real
+workflows, where a tool succeeds (exit 0) but emits a `…-failed` marker and so every
+such marker had to be hand-routed to one shared handler, one edge at a time. `else`
+states "everything not handled above goes here" once.
+
+- **At most one `else` per edges block.** A second is a parse error.
+- **`else -> ` requires a target node.** A bare `else ->` with no destination is a parse error.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)